README.md

```diff
@@ -245,44 +192,24 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project feel free to leave a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
```

Comment:
- [ ] The acknowledgement section is still blank. As informed in the previous review let's endeavor to add some details to this section. You can choose to acknowledge for instance React.js or Microverse. Doing so makes the ReadMe appear more professional! 😉 

**The acknowledgement section is blank:**
![image](https://user-images.githubusercontent.com/12745474/222759923-05a5073b-f4ed-4809-a9bb-e5d016269f1e.png)